trillian/ctfe/handlers.go

```diff
@@ -46,7 +46,10 @@ import (
 )
 
 // TODO(drysdale): remove this flag once everything has migrated to ByRange
```

Comment:
This comment should be moved to follow the flag it's referencing?

Reply:
done